lib/node_modules/@stdlib/math/special/sin/scripts/config.js

```diff
@@ -0,0 +1,38 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MAIN //
+
+/**
+* Configuration object.
+*
+* @private
+* @type {Object}
+*/
+var config = {
+	'input_dtypes': 'real_and_generic',
+	'output_dtypes': 'real_floating_point',
```

Comment:
Here, we should also allow `real_floating_point_and_generic`. Meaning, if I give a "generic" array, I should get a "generic" array out, by default.